tests/BoltOn.Tests/UoW/UnitOfWorkManagerTests.cs

```diff
@@ -101,12 +101,12 @@ public void Get_WithDefaultsTwice_StartsTransactionWithDefaultsAndJoinsWithThePr
 			var result2 = sut.Get(unitOfWorkOptions);
 
 			// assert
-			var uowProviderLoggerStmt = $"About to start UoW. IsolationLevel: {IsolationLevel.Serializable} " +
+			var uowProviderLoggerStmt = $"About to start UoW. IsolationLevel: {IsolationLevel.ReadCommitted} " +
```

Comment:
this unit test's method name needs to be updated

Reply:
@gokulm I feel this test is redundant, and also a duplicate of next test at line #119. What do you think?

Reply:
@cthumburu agree. #119 test's RequiresNew can be removed though